server/odc-service/src/main/java/com/oceanbase/odc/service/session/factory/DruidDataSourceFactory.java

```diff
@@ -59,7 +62,9 @@ public DataSource getDataSource() {
         String jdbcUrl = getJdbcUrl();
         String username = getUsername();
         String password = getPassword();
-        DruidDataSource dataSource = new InnerDataSource(new SessionCreatedInitializer(connectionConfig));
+        DruidDataSource dataSource =
+                new InnerDataSource(Arrays.asList(new DataSourceInitScriptInitializer(connectionConfig),
```

Comment:
we may add BackupInstanceInitializer first and add DataSourceInitScriptInitializer second

Reply:
Sure~